src/hooks/motionWidgets/stakingWidget/helpers.ts

```diff
@@ -0,0 +1,37 @@
+import { StakingSliderProps } from '~common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/StakingWidget/StakingSlider/StakingSlider';
+
+export const mapStakingSliderProps = ({
+  isObjection,
+  minUserStake,
+  remainingToStake,
+  canBeStaked,
+  maxUserStake,
+  userActivatedTokens,
+  nativeTokenDecimals,
+  nativeTokenSymbol,
+  totalPercentage,
+  enoughTokens,
+  reputationLoading,
+  getErrorType,
+  mutableRef,
+}): StakingSliderProps => ({
```

Comment:
Same as I mentioned above. Can we pass the whole motion data directly, as destructuring individual keys is going to cause trouble in the future